export-samples/html/cv-monochrome-force.html

```diff
@@ -1032,13 +1037,18 @@ <h2 class="title">Educación</h2>
 
         if (window.innerWidth > 834) {
           // Mueve los elementos a las columnas correspondientes
```

Comment:
Not sure if we are commenting in english or spanish, if it's english move it to english

Reply:
removed comments, not neccesary